src/client/content_mapblock.cpp

```diff
@@ -208,14 +208,8 @@ void MapblockMeshGenerator::drawCuboid(const aabb3f &box,
 		video::S3DVertex(min.X, min.Y, min.Z, 0, 0, -1, colors[5], txc[20], txc[23]),
 	};
 
-	static const u8 light_indices[24] = {
-		3, 7, 6, 2,
-		0, 4, 5, 1,
-		6, 7, 5, 4,
-		3, 2, 0, 1,
-		7, 3, 1, 5,
-		2, 6, 4, 0
-	};
```

Comment:
Another instance of clang-format messing with code containing "logically-sane" line-breaks.

Reply:
Yes, it is going to mess up a lot of stuff like this and make it unreadable.

Reply:
If you configure it right, these things should be rare and minimal. :)